app/helpers/support_helper.rb

```diff
@@ -1,4 +1,15 @@
 module SupportHelper
+
+  def time_until_expiration(subscription_session)
+    minutes = (subscription_session.expires_at.to_i - Time.current.to_i) / 60
+
+    if minutes == 0
```

Comment:
`if minutes.zero?`

Reply:
Yeah, Rubocop barked at me about that. Just haven't push it up yet.